### DISCUSSION TITLE
Add a mod

### DISCUSSION THREAD

**Discussion by doemty** [UltimatChamp/optimum-fuse#12] [Ideas]
I suggest add mod "Don't Clear Chat History".
As the title says.
https://modrinth.com/mod/dcch

**Comment by UltimatChamp** [comment]
Can you file an issue regarding this because we have an issue template for mod suggestions now 😄

**Reply by UltimatChamp** [reply]
https://github.com/UltimatChamp/optimum-fuse/issues

**Comment by UltimatChamp** [comment]
oh! I can do this manually
